p03-form-validations/src/Shared/components/Button/Button.component.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react'
 import "./Button.styles.css"
 
-const Button = ({ title }) => {
+const Button = ({ title, handleClick, isDisabled, className } ) => {
```

Comment:
there is not problem with this, just to give you a tip, may be the handleClick property of the button, could be renamed to onClick, this way it could be easier to remember, because htmlElements has the onClick function and the isDisabled could be disabled, in order to do a spread operator to the button with the variables
 
 ```
 const Button = ({ title, ...props }) => <button {...props}>{title}</button>
 ```